tests/stubs.py

```diff
@@ -24,22 +27,49 @@ def apdu(*args):
     apdu = bytes.fromhex(request.get_json().get("data", "00"))
     status = f"{APDUStatus.SUCCESS:x}" if apdu[0] == 0x00 else f"{APDUStatus.ERROR:x}"
     return {"data": EndPoint.APDU + status}
-def button(*args):
-    return {}
+
+class Actions:
+    def __init__(self):
+        self.iter = 0
+        self.idx = 1
+    
+    def button(self,*args):
+        path = request.path
+        if "right" in path:
+            self.idx = self.idx+1 if (self.idx < 2) else 0
+        elif "left" in path:
+            self.idx = self.idx-1 if (self.idx >= 1) else 0
+        elif "both" in path:
+            if self.idx==2:
```

Comment:
There are a lot of inconsistency with spaces before/after operators, comas, etc. Maybe we should add `yapf` on tests too.

Reply:
Done.